### DISCUSSION TITLE
How to do Hotkeys?

### DISCUSSION THREAD

**Discussion by Kreijstal** [nut-tree/nut.js#421] [Q&A] [answered]
How do I do hotkeys, when I press a key combination it does something?

**Comment by s1hofmann** [comment, marked as answer]
Hi @Kreijstal 👋 

Do you want to trigger a hotkey combination with nut.js, or are you looking for a way to programmatically register global hotkey combinations that execute e.g. a nut.js automation?

The former is easily possible via the nut.js [keyboard API](https://nutjs.dev/API/keyboard), the latter is not supported by nut.js.
You'd have to use a different package like [iohook](https://www.npmjs.com/package/iohook?activeTab=readme)

**Reply by Kreijstal** [reply]
yes, I wanted to programmatically execute something with nut.js ala ahk as soon as I pressed certain key combinations, but it's not possible, saaaad thanks for showing me the way
